src/util/MemorySize/MemorySize.h

```diff
@@ -97,6 +101,24 @@ class MemorySize {
   */
   static MemorySize parse(std::string_view str);
 
+  // Arithmetic operators and arithmetic assignment operators.
+  constexpr MemorySize operator+(const MemorySize& m) const;
+  constexpr MemorySize& operator+=(const MemorySize& m);
+  constexpr MemorySize operator-(const MemorySize& m) const;
+  constexpr MemorySize& operator-=(const MemorySize& m);
+
+  template <Arithmetic T>
+  constexpr MemorySize operator*(const T c) const;
+
+  template <Arithmetic T>
+  constexpr MemorySize& operator*=(const T c);
+
+  template <Arithmetic T>
+  constexpr MemorySize operator/(const T c) const;
```

Comment:
For the sake of completeness:
I think this allows `3_MB * 2`, but not `2 * 3_MB`. Can you also add those?
this is best done by either a free function (outside the class, but in the same namespace), 
or as a `friend` function.

Reply:
Oh, I thought that would have been done automatically. My bad. Fixed in [7b640a9](https://github.com/ad-freiburg/qlever/pull/1079/commits/7b640a988b3ef8a7e51a068374d6c5fcd5ec7580).